client/src/modules/stock/aggregated_consumption/aggregated_consumption.js

```diff
@@ -249,6 +269,37 @@ function StockAggregatedConsumptionController(
       user : Session.user.display_name,
     };
 
+    const checkOverconsumption = vm.Stock.store.data;
+
+    checkOverconsumption.forEach(stock => {
+      stock.quantityAvailable = 0;
+
+      vm.currentInventories.forEach(lot => {
+        if (lot.uuid === stock.uuid) {
+          stock.quantityAvailable = lot.quantity;
+        }
+      });
+    });
+
+    vm.overconsumption = checkOverconsumption.filter(
+      c => (c.quantity_consumed + c.quantity_lost) > c.quantityAvailable,
+    );
+
+    if (vm.overconsumption.length) {
+      vm.overconsumption.forEach(item => {
+        item.textI18n = {
+          text : item.text,
+          label : item.label,
+          quantityAvailable : item.quantityAvailable,
+          quantity : (item.quantity_consumed + item.quantity_lost),
+        };
+      });
+
+      Notify.danger('ERRORS.ER_PREVENT_NEGATIVE_QUANTITY_IN_STOCK');
```

Comment:
This error message gives no clue regarding which stock item is causing the problem.  I think it would be helpful to give the stock item name/description in the error message.   Unless the next error message has those details?

Reply:
Indeed there is a message that describes the products that are problematic
![image](https://user-images.githubusercontent.com/8035588/112827194-ead7e680-9085-11eb-9303-6d5171fbf0f7.png)

Reply:
Good to see this message with the details.